age/keysource.go

```diff
@@ -233,6 +238,98 @@ func (key *MasterKey) TypeToIdentifier() string {
 	return KeyTypeIdentifier
 }
 
+// readPublicKeyFile attempts to read a public key based on the given private
+// key path. It assumes the public key is in the same directory, with the same
+// name, but with a ".pub" extension. If the public key cannot be read, an
+// error is returned.
+func readPublicKeyFile(privateKeyPath string) (ssh.PublicKey, error) {
```

Comment:
This function is very similar to `readPubFile()` in https://github.com/FiloSottile/age/blob/v1.0.0/cmd/age/parse.go#L231-L253. How about moving this function (and the one below) into a new file `ssh_parse.go`, which has a similar starting commit than `tui.go`? That way we avoid any potential problems with licensing and copyright.

Reply:
Sounds reasonable to me. done.